src/index.js

```diff
@@ -3,8 +3,24 @@ import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
 import * as serviceWorker from './serviceWorker';
+import { IntlProvider } from 'react-intl';
+import messages_es from './i18n/es.json';
+import messages_en from './i18n/en.json';
 
-ReactDOM.render(<App />, document.getElementById('root'));
+const messages = {
+  es: messages_es,
+  en: messages_en,
+};
+
+//TODO: this hardcoded data will depend by the app language
+const language = navigator.language.toLowerCase().split(/[_-]+/)[0] || 'en';
```

Comment:
Yes,  it could be a possibility. I didn't think of that. Maybe we can also use the default messages that the library provides when it doesn't find the language.